pennylane_lightning_gpu/src/tests/mpi/Test_MPI_StateVectorCudaManaged_NonParam.cpp

```diff
@@ -0,0 +1,333 @@
+#include <algorithm>
+#include <complex>
+#include <iostream>
+#include <limits>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include <catch2/catch.hpp>
+#include <mpi.h>
+
+#include "cuGateCache.hpp"
+#include "cuGates_host.hpp"
+#include "cuda_helpers.hpp"
+
+#include "StateVectorCudaMPI.hpp"
+#include "StateVectorCudaManaged.hpp"
+#include "StateVectorRawCPU.hpp"
+
+#include "MPIManager.hpp"
+
+#include "../TestHelpers.hpp"
+
+using namespace Pennylane;
+using namespace Pennylane::MPI;
+using namespace CUDA;
+
+#define num_qubits 8
+#define lsb_1qbit                                                              \
+    { 0 }
+#define msb_1qbit                                                              \
+    { num_qubits - 1 }
+#define lsb_2qbit                                                              \
+    { 0, 1 }
+#define msb_2qubit                                                             \
+    { num_qubits - 2, num_qubits - 1 }
+#define mlsb_2qubit                                                            \
+    { 0, num_qubits - 1 }
+#define lsb_3qbit                                                              \
+    { 0, 1, 2 }
+#define msb_3qubit                                                             \
+    { num_qubits - 3, num_qubits - 2, num_qubits - 1 }
+#define mlsb_3qubit                                                            \
+    { 0, num_qubits - 2, num_qubits - 1 }
```

Comment:
What `lsb`, `msb` and `mlsb` mean?
Can we have more descriptive names?

Reply:
In bit [numbering,](https://en.wikipedia.org/wiki/Bit_numbering) , the least significant bit (LSb) is the [bit](https://en.wikipedia.org/wiki/Bit) position in a [binary](https://en.wikipedia.org/wiki/Binary_numeral_system) [integer](https://en.wikipedia.org/wiki/Integer_(computer_science)) representing the binary 1s place of the integer. Similarly, the most significant bit (MSb) represents the highest-order place of the binary integer.